.github/FUNDING.yml

```diff
@@ -0,0 +1,3 @@
+# These are supported funding model platforms
+
+custom: ["https://www.kitware.eu/contact-us/", kitware.eu]
```

Comment:
```suggestion
custom: ["https://www.kitware.eu/contact-us/", "https://kitware.eu"]
```

While not strictly required [^1], I suggest being consistent with the use of `"`.

For reference, here is a prior experiment[^2] showing how the links will render (with or without explicitly specifying the `https://` for the second link).

![image](https://user-images.githubusercontent.com/219043/206002161-53abf2bc-4360-4341-9afe-e3d28226d55a.png)


[^1]: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository#about-funding-files

[^2]: https://github.com/jcfr/ci-sandbox/commit/c0a9060749074598eda2b32775d45c2fea7c1d50

Reply:
Thanks !

I left it without the "https://" prefix, just like [vtk.js](https://github.com/Kitware/vtk-js/blob/master/.github/FUNDING.yml).